testing/test_pluginmanager.py

```diff
@@ -188,7 +188,7 @@ def test_traceback():
     with pytest.raises(ImportError) as excinfo:
         pytestpm.import_plugin("qwe")
 
-    assert str(excinfo.value).endswith(
+    assert str(excinfo.value).find(
```

Comment:
find is not a helpful assertion:

```pycon
>>> assert 'watwatwat'.find('asdf')
>>>
```

Reply:
Yes, the result of test with ``find`` confused me.